src/PlatformHelpers.native.js

```diff
@@ -1,5 +1,13 @@
 /* @flow */
 
-import { BackAndroid, Linking } from 'react-native';
+import {
+  BackAndroid as OldBackAndroid,
+  BackHandler as NewBackAndroid,
+  Linking,
+} from 'react-native';
 
-export { BackAndroid, Linking };
+const fullVersion = require('react-native/package.json').version.split('.')[1];
+
+let BackHandler = fullVersion < 44 ? OldBackAndroid : NewBackAndroid;
```

Comment:
just look for BackHandler and use it if available otherwise use BackAndroid. no need to check for RN version